findbugs-exclude.xml

```diff
@@ -0,0 +1,5 @@
+<FindBugsFilter>
+     <Match>
+       <Class name="~.+\.Messages" />
```

Comment:
I created a new plugin project, added a `Messages.properties` with

```properties
HelloWorldBuilder.bonjour_=Bonjour, {0}!
HelloWorldBuilder.hello_=Hello, {0}!
```

as in

```java
        if (getDescriptor().getUseFrench())
            listener.getLogger().println(Messages.HelloWorldBuilder_bonjour_(name));
        else
            listener.getLogger().println(Messages.HelloWorldBuilder_hello_(name));
```

and did a `mvn clean install`. Passed with

```
--- findbugs-maven-plugin:3.0.3:check (findbugs) @ localizer-findbugs ---
BugInstance size is 0
Error size is 0
No errors/warnings found
```

`Messages.java` started with

```java
@SuppressWarnings({
    "",
    "PMD",
    "all"
})
```

though this was with `SOURCE` retention so I cannot imagine that it affected FindBugs.